app/models/solidus_stripe/payment_method.rb

```diff
@@ -4,40 +4,153 @@ module SolidusStripe
   class PaymentMethod < ::Spree::PaymentMethod
     preference :api_key, :string
     preference :publishable_key, :string
+    preference :webhook_endpoint_secret, :string, default: ''
```

Comment:
Unless I'm missing something that's not used anywhere yet. https://github.com/solidusio/solidus_stripe/issues/161 will handle it, so I think we should remove this line for now.